app/services/gplaces.js

```diff
@@ -17,5 +17,32 @@ module.exports = function() {
           });
     });
   };
+  this.getPlaceByText = (query, optionalParameters)=>{
+    return new Promise((resolve, reject)=>{
```

Comment:
Spaces hinzufügen (also `) => {`)

Reply:
So ist das nicht in der lint config definiert, das ist so in Ordnung